src/main/java/com/minecolonies/coremod/achievements/MineColoniesAchievement.java

```diff
@@ -13,35 +13,33 @@
  *
  * @since 0.2
  */
-public class MineColoniesAchievement extends Achievement
+public class MineColoniesAchievement
 {
-
+    //todo this will make new advancements at some point
     /**
      * Create a new Achievement.
-     *
-     * @param id       the unique id this achievement should have
+     *  @param id       the unique id this achievement should have
      * @param offsetX  X position placement on the board
      * @param offsetY  Y position placement on the board
      * @param itemIcon The icon to show
      * @param parent   the parent achievement
      */
-    public MineColoniesAchievement(final String id, final int offsetX, final int offsetY, final Item itemIcon, final Achievement parent)
+    public MineColoniesAchievement(final String id, final int offsetX, final int offsetY, final Block itemIcon, final MineColoniesAchievement parent)
     {
-        super(Constants.MOD_ID + "." + id, id, offsetX, offsetY, itemIcon, parent);
+        //super(Constants.MOD_ID + "." + id, id, offsetX, offsetY, itemIcon, parent);
     }
 
     /**
      * Create a new Achievement.
-     *
-     * @param id        the unique id this achievement should have
+     *  @param id        the unique id this achievement should have
      * @param offsetX   X position placement on the board
      * @param offsetY   Y position placement on the board
      * @param blockIcon The icon to show
      * @param parent    the parent achievement
      */
-    public MineColoniesAchievement(final String id, final int offsetX, final int offsetY, final Block blockIcon, final Achievement parent)
+    public MineColoniesAchievement(final String id, final int offsetX, final int offsetY, final Item blockIcon, final MineColoniesAchievement parent)
     {
-        super(Constants.MOD_ID + "." + id, id, offsetX, offsetY, blockIcon, parent);
+        //super(Constants.MOD_ID + "." + id, id, offsetX, offsetY, blockIcon, parent);
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3ACommentedOutCodeLine)